content/learn/book/getting-started/ecs/_index.md

```diff
@@ -144,3 +144,34 @@ hello Zayna Nieves!
 Marvelous!
 
 **Quick Note**: "hello world!" might show up in a different order than it does above. This is because systems run in parallel by default whenever possible.
+
+If we want to change the names of some people (perhaps they got married!), for example, we can do this using a mutable query:
+
+```rs
+fn update_people(mut query: Query<&mut Name, With<Person>> {
+    for mut name in &mut query {
+        if name.0 == "Elaine Proctor" {
+            name.0 = "Elaine Hume".to_string();
+            break; // We break out of the loop here, because we don’t need to change any other names
+        }
+    }
+}
+```
+
+The `query` argument must be mutable and the components we want to change have to be a mutable reference (`&mut`) for this to work.
```

Comment:
I think this might be more readable. What do you think?
```suggestion
We need to make `query` mutable, and use a mutable reference (`&mut`) to the components we want to change.
```